test/Casting/Casts.swift

```diff
@@ -924,4 +924,27 @@ CastsTests.test("Optional cast to AnyHashable") {
   expectNotEqual(xh, yh)
 }
 
+// Repeatedly casting to AnyHashable should still test equal.
+// (This was broken for a while because repeatedly casting to
+// AnyHashable could end up with multiple nested AnyHashables.)
+// rdar://75180619
+CastsTests.test("Recursive AnyHashable") {
```

Comment:
We'll need to have a `guard #available(macOS 9999, yada yada, *)` here because the conformance won't be there in back deployment tests.

Reply:
This entire test suite is `UNSUPPORTED: use_os_stdlib` already.